R/pkg/R/mllib.R

```diff
@@ -695,17 +695,15 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
           function(data, layers, blockSize = 128, solver = "l-bfgs", maxIter = 100,
-                   tol = 1E-6, stepSize = 0.03, seed = 0x7FFFFFFF) {
+                   tol = 1E-6, stepSize = 0.03, seed = "") {
```

Comment:
this is a bit unusual in R, also could cause confusion as to what should be passed in (string? integer?) - how about we default to NULL and then check inside spark.mlp to map seed NULL to "" (only that)?

also if seed is not NULL it should be coerce into integer (as.integer(seed))